config.py

```diff
@@ -2,15 +2,15 @@
 
 
 class Config(object):
-    CONTACT_EMAIL = os.environ.get("CONTACT_EMAIL")
-    CONTACT_PHONE = os.environ.get("CONTACT_PHONE")
-    DEPARTMENT_NAME = os.environ.get("DEPARTMENT_NAME")
-    DEPARTMENT_URL = os.environ.get("DEPARTMENT_URL")
+    CONTACT_EMAIL = os.environ.get("CONTACT_EMAIL", "")
```

Comment:
so the non e2e tests run without having to set these as we dont care about them in the current tests. maybe at some point we will override these in tests but not an issue for now